scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/dfn.yaml

```diff
@@ -297,6 +297,38 @@ DFN-8-1EP_3x3mm_P0.65mm_EP1.7x2.05mm:
   #include_suffix_in_3dpath: 'False'
   #include_pad_size: 'fp_name_only' # 'both' | 'none' (default)
 
+DFN-10-1EP_3x3mm_P0.5mm_EP1.7x2.5mm:
+  device_type: 'DFN'
+  #manufacturer: 'man'
+  #part_number: 'mpn'
+  size_source: 'https://www.monolithicpower.com/pub/media/document/MPQ2483_r1.05.pdf'
+  # ipc_class: 'qfn' # 'qfn_pull_back'
+  #ipc_density: 'least' #overwrite global value for this device.
+  body_size_x:
+    minimum: 2.9
+    maximum: 3.1
+  body_size_y:
+    minimum: 2.9
+    maximum: 3.1
+  lead_width:
+    minimum: 0.18
+    maximum: 0.3
+  lead_len:
+    minimum: 0.3
+    maximum: 0.5
+
+  EP_size_x: 1.7
+  EP_size_y: 2.5
```

Comment:
Add the ep size as specified with minimum and maximum and use the EP_size_x_overwrite (and y) parameters to force a specific pad size.

```yaml
EP_size_x:
  minimum: 1.45
  maximum: 1.75
EP_size_x_overwrite: 1.7
```
(Note that the overwrite is a new parameter not intended under the normal ep size parameters)

As an example look at the size definition for `DFN-8-1EP_2x2mm_P0.5mm_EP0.8x1.2mm`

Reply:
Done now, thanks!